www/includes/easyparliament/templates/html/people/mp_index.php

```diff
@@ -0,0 +1,226 @@
+                    <?php } else { ?>
+                    <li><strong>Sorted by</strong> Last name</li>
+                    <li>Sort by <a href="<?= $urls['by_first'] ?>">First name</a> / <a href="<?= $urls['by_party'] ?>">Party</a></li>
+                    <?php } ?>
+                </ul>
+                <?php } else { ?>
+                <ul class="search-result-display-options">
+                    <?php if ( $order == 'party' ) { ?>
+                    <li><strong>Sorted by</strong> Party</li>
+                    <li>Sort by <a href="<?= $urls['by_name'] ?>">Name</a></li>
+                    <?php } else { ?>
+                    <li><strong>Sorted by</strong> Name</li>
+                    <li>Sort by <a href="<?= $urls['by_party'] ?>">Party</a></li>
+                    <?php } ?>
+                </ul>
+                <?php } ?>
+
+                <div class="people-list-alphabet">
+                    <a href="#A">A</a>
+                    <a href="#B">B</a>
+                    <a href="#C">C</a>
+                    <a href="#D">D</a>
+                    <a href="#E">E</a>
+                    <a href="#F">F</a>
+                    <a href="#G">G</a>
+                    <a href="#H">H</a>
+                    <a href="#I">I</a>
+                    <a href="#J">J</a>
+                    <a href="#K">K</a>
+                    <a href="#L">L</a>
+                    <a href="#M">M</a>
+                    <a href="#N">N</a>
+                    <a href="#O">O</a>
+                    <a href="#P">P</a>
+                    <a href="#Q">Q</a>
+                    <a href="#R">R</a>
+                    <a href="#S">S</a>
+                    <a href="#T">T</a>
+                    <a href="#U">U</a>
+                    <a href="#V">V</a>
+                    <a href="#W">W</a>
+                    <a href="#X">X</a>
+                    <a href="#Y">Y</a>
+                    <a href="#Z">Z</a>
+                </div>
+
+                <div class="people-list">
+                <?php $current_initial = ''; ?>
+                <?php foreach ( $data as $person ) {
+                    $initial = substr( strtoupper($person['family_name']), 0, 1);
+                    if ( $initial != $current_initial ) {
+                        $current_initial = $initial;
+                        $initial_link = "name=\"$initial\" ";
+                    } else {
+                        $initial_link = "";
+                    }
+                ?>
+                <a <?= $initial_link ?>href="/mp/<?= $person['url'] ?>" class="people-list__person">
+                <img class="people-list__person__image" src="<?= $person['image'] ?>">
+                        <h2 class="people-list__person__name"><?= $person['name'] ?></h2>
+                        <p class="people-list__person__memberships">
+                        <span class="people-list__person__constituency"><?= $person['constituency'] ?></span>
```

Comment:
On /peers, all the parties are indented because of the blank constituency, not sure if that's wanted or not.